### PR TITLE
fix: make annoucement filed in speakers call nullable

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5578

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: migrations/versions/rev-2020-11-12-09:02:31-aa9daed401f7_.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = 'aa9daed401f7'
+down_revision = '5e7082330d80'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('speakers_calls', 'announcement',
```
Comment (iamareebjamal) [inline migrations/versions/rev-2020-11-12-09:02:31-aa9daed401f7_.py:21]: After this line, please add statement for converting empty string to null as well

File: app/api/schema/speakers_calls.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def validate_date(self, data, original_data):
         #                               "speakers-call ends-at should be before event starts-at")
 
     id = fields.Str(dump_only=True)
-    announcement = fields.Str(required=True)
+    announcement = fields.Str(required=False)
```
Comment (iamareebjamal) [inline app/api/schema/speakers_calls.py:57]: Should be allow_none=True

**Comment by iamareebjamal** [comment]
Missing https://github.com/fossasia/open-event-server/pull/7435#discussion_r521813900

**Comment by maze-runnar** [comment]
> Missing [#7435 (comment)](https://github.com/fossasia/open-event-server/pull/7435#discussion_r521813900)

yes, I don't exactly know how to do that. Trying to find

**Comment by iamareebjamal** [comment]
`op.execute("update speaker_calls set announcement = null where announcement = ''");`

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**